### PR TITLE
Migrate nightly JDKx to GHActions

### PR DESCRIPTION
**Pull request by ignasi35**
refs https://github.com/lightbend/akka-platform-meta/issues/253

Built on top of #30449

### CODE REVIEW

File: .github/workflows/nightly-jdk11.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Nightly jkd11
+
+on:
+  schedule:
+    - cron:  '0 6 * * *'
```
Comment (andreaTP) [inline .github/workflows/nightly-jdk11.yml:5]: I don't think we need to separate the corn jobs now, they can run at the same time

File: .github/workflows/nightly-jdk8-scala2.13.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Nightly jdk8/scala2.13
```
Comment (andreaTP) [inline .github/workflows/nightly-jdk8-scala2.13.yml:1]: can this be merged with the `Nightly jdk8` ?

File: .github/workflows/nightly-jdk11.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  compile-and-test:
+    name: Compile and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
```
Comment (andreaTP) [inline .github/workflows/nightly-jdk11.yml:17]: I think we can squash all the nightly in one single file and have separate jobs depending on the JDK, or even a single job switching JDK in the middle.

Comment (andreaTP) [reply .github/workflows/nightly-jdk11.yml:17]: (to at least re-use the scheduling and mailing boilerplate)

**Comment by raboof** [comment]
(looking into the TlsSpec failure)

**Comment by patriknw** [comment]
Superseded by https://github.com/akka/akka/pull/30521

**Pull request closed without merging**